src/server/models/organisation.js

```diff
@@ -0,0 +1,34 @@
+import SequelizeSlugify from 'sequelize-slugify';
+import { DataTypes } from 'sequelize';
+
+import db from '~/server/database';
+
+const Organisation = db.define('organisation', {
+  id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  slug: {
+    type: DataTypes.STRING,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  description: {
+    type: DataTypes.STRING(2000),
+    allowNull: true,
+  },
+  url: {
```

Comment:
I don't think we need this field (and there's no corresponding column on the table in the db)